core-primitives/sgx-runtime-primitives/src/constants.rs

```diff
@@ -0,0 +1,12 @@
+use crate::types::{BlockNumber, Moment};
```

Comment:
Can you add a header here?

Reply:
Added it in b1374ecce5541b9e799cca11b9b63de2bc581b30.